Source/WebKit/Shared/unix/BreakpadExceptionHandler.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * If not stated otherwise in this file or this component's license file the
+ * following copyright and licenses apply:
```

Comment:
It's almost as if whoever wrote this text designed it to be maximally confusing. I think you're supposed to remove this when copying code from wherever this used to live to someplace new. Otherwise, the implication is that you can ignore the Apache license and apply WebKit's BSD plus LGPLv2+ license instead, which probably isn't right?

Note that Apache is not actually a permitted license for WebKit, but we do already use it in a few places, so adding a little more doesn't make much difference. I'm unsure what to do about that.